src/foam/u2/ExportModal.js

```diff
@@ -144,6 +169,9 @@ foam.CLASS({
         document.body.appendChild(link);
         link.click();
       })
+
+      if ( this.exportAllColumns )
+        this.filteredTableColumns$.set(filteredColumnsCopy);
```

Comment:
Couldn't you just do this.filteredTableColumns = null; ?

Reply:
fixed